doc/builders.rst

```diff
@@ -76,6 +76,22 @@ The builder's "name" must be given to the **-b** command-line option of
 
    .. _Qt help: http://qt-project.org/doc/qt-4.8/qthelp-framework.html
 
+.. module:: sphinx.builders.applehelp
+.. class:: AppleHelpBuilder
+
+   This builder produces an Apple Help Book based on the same output as the
+   standalone HTML builder.
+
+   This builder requires Mac OS X 10.6 or higher to function, as it depends on
```

Comment:
E.g. for the Windows HTML help builder, we have omitted calling the "help compiler" from Sphinx and added that to the instructions of things to do. That way, people can use it somewhere else or use a non-standard (i.e. Wine) installation. Would it make sense to do a similar thing here?